### PR TITLE
Various updates from a fresh deployment

### PR DESCRIPTION
**Pull request by dmsimard**
Each commit should be fairly self-explanatory but in summary:
- Updated configuration parameters in ansibullbot.cfg
- Don't try to use slack if it's not configured
- Cleaned up shippable configuration that is no longer used

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
   community.general.slack:
     msg: "{{ ansibullbot_slack_message }}"
     token: "{{ ansibullbot_slack_token }}"
+  when: ansibullbot_slack_token != []
```
Comment (samdoran) [inline handlers/main.yml]:
```suggestion
  when: ansibullbot_slack_token is truthy
```

**Review by samdoran — CHANGES_REQUESTED** [review]
These are great changes. Just one minor suggestion.

File: handlers/main.yml (hunk 1 above)
Comment (dmsimard) [reply handlers/main.yml]: I didn't even know ``truthy`` was a test. TIL.

I like it. Thanks !

**Review by samdoran — APPROVED** [review]

**Comment by samdoran** [comment]
Thank you for the changes!

**Pull request merged**